addons/project/static/src/components/project_right_side_panel/components/project_profitability.js

```diff
@@ -12,8 +12,8 @@ export class ProjectProfitability extends Component {
     }
 
     get margin() {
-        const invoiced_billed = this.revenues.total.invoiced - this.costs.total.billed;
-        const to_invoice_to_bill = this.revenues.total.to_invoice - this.costs.total.to_bill;
+        const invoiced_billed = this.revenues.total.invoiced + this.costs.total.billed;
```

Comment:
Hello @mamatch 👋🏼 
Thanks for the fix 🙂 
Could you just rephrase you commit title like this?
`[FIX] project: ensure cost are subtracted when calculating margin`
As per guidelines it should start with a verb see [here](https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html#commit-message-header).

Reply:
Done :)